examples/hello-triangle/main.rs

```diff
@@ -60,7 +62,7 @@ async fn run() {
         }),
         primitive_topology: wgpu::PrimitiveTopology::TriangleList,
         color_states: &[wgpu::ColorStateDescriptor {
-            format: wgpu::TextureFormat::Bgra8UnormSrgb,
+            format: wgpu::TextureFormat::Bgra8Unorm,
```

Comment:
Oh interesting. Is it now the case that `Bgra8Unorm` is assumed to be sRGB?

Reply:
Nope both `Bgra8Unorm` and `Bgra8UnormSrgb` still exist – this is just a temporary hack for the hello-triangle example because Chrome Canary didn't seem to allow me to use `Bgra8UnormSrgb` for the swapchain :) I haven't looked into it yet but a swapchain with `Bgra8UnormSrgb` is supposed to work fine in browsers as far as I know.